strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java

```diff
@@ -56,6 +58,18 @@
     @Inject
     private List<AuthenticationSupplier> suppliers;
 
+    
+    
+    @Override
+    public void init(WebSecurity web)
+        throws Exception
+    {
+        super.init(web);
+        DefaultHttpFirewall httpFirewall = new DefaultHttpFirewall();
```

Comment:
For my own understanding -- is this part here a solution to that issue you were having, or is it a workaround?

Reply:
I have dilabled encoded `/`.I think that `DirectoryTraversalFilter` should check encoded slashes instead of this default firewall.

Reply:
So, is there still anything that needs to be done about this, or is it all good now?
:)

Reply:
We need to check `DirectoryTraversalFilter` to be sure that it resolves encoded `/` correctly

Reply:
Would you mind raising a new task with a proper description for it, so that we could follow-up on it when we get some more time? Thanks! :)